keras/ops/core.py

```diff
@@ -204,6 +204,16 @@ def slice_update(inputs, start_indices, updates):
     return backend.core.slice_update(inputs, start_indices, updates)
 
 
+@keras_export("keras.ops.scan")
```

Comment:
There should be a correspond symbolic op class.

Also, please add a docstring for this function.

Reply:
done.I am not sure about the corresponding symbolic class